.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+StringLiterals:
+  Enabled: false
```

Comment:
The second part will also take care of some of the complaints that remained in the RuboCop output.